support-frontend/assets/pages/digital-subscription-landing/components/paymentSelection/helpers/paymentSelection.jsx

```diff
@@ -138,7 +138,11 @@ const mapStateToProps = (state: State): PropTypes => {
         title: BILLING_PERIOD_GIFT[digitalBillingPeriodGift].title,
         price: getDisplayPrice(currencyId, fullPrice),
         href: getDigitalCheckout(countryGroupId, billingPeriodForHref, promoCode, orderIsAGift),
-        onClick: sendTrackingEventsOnClick('subscribe_now_cta_gift', 'DigitalPack', null, billingPeriod),
+        onClick: sendTrackingEventsOnClick({
+          id: `subscribe_now_cta_gift-${billingPeriod}`,
+          product: 'DigitalPack',
+          componentType: 'ACQUISITIONS_BUTTON',
+        }),
```

Comment:
Updated the `id` to include `billingPeriod` so we can get more info on the CTA selected. Corresponding Trello Card [here](https://trello.com/c/ynLFxBjM/3567-update-click-tracking-component-id-on-pricing-cards-on-the-digital-subs-paper-product-landing-pages-to-illustrate-period).